snapshot.css

```diff
@@ -1340,11 +1453,11 @@ html:lang(ar)
   flex-direction: row;
   align-items: center;
   justify-content: flex-start;
-  margin: 0 -0.375rem;
+  margin: 0 -var(--kz-var-spacing-sm, 0.75rem) / 2;
```

Comment:
This is an error:
Should be fixed in `draft-packages-filter-menu-button-KaizenDraft-FilterMenuButton-styles-module__filterMenuButton--3RBsO`